src/io/ipc/read/deserialize.rs

```diff
@@ -7,6 +7,7 @@ use arrow_format::ipc::MetadataVersion;
 use crate::array::*;
 use crate::datatypes::{DataType, Field, PhysicalType};
 use crate::error::Result;
+use crate::io::ipc::read::common::ReadBuffer;
```

Comment:
I have been trying to make `common` private - could we pub export this at `io::ipc::read` so users can just import from it?

Reply:
Will do. :+1: